content/en/docs/tutorials/hello-minikube.md

```diff
@@ -143,6 +143,8 @@ recommended way to manage the creation and scaling of Pods.
     ```
 
 1. View application logs for a container in a pod.
+   Make sure to replace `hello-node-5f76cf6ccf-br9b5` with the name of the pod from
+   `kubectl get pods` command output.
```

Comment:
Hi @digininja .
What do you think about adding a note?

```
{{< note >}}
Replace `hello-node-5f76cf6ccf-br9b5` in the `kubectl logs` command with the name of the pod from the `kubectl get pods` command output.
{{< /note >}}
```

Reply:
@digininja , what do you think of @kbhawkey 's [suggestion](https://github.com/kubernetes/website/pull/44115/files#r1434631864)?

Reply:
Yes, it looks fine to me.

Reply:
@digininja,

> Yes, it looks fine to me.

Are you still working on @kbhawkey 's [suggestion](https://github.com/kubernetes/website/pull/44115/files#r1434631864)?
Please push the changes to PR to proceed further.